### PR TITLE
[New] Snap arm support

### LINKED ISSUE #5178: Snap: Newest releases for ARM architectures - Raspberry Pi
iOS mobile application unable to connect to raspberry pi ubuntu.

Cites: 

**The server ***http://myserver.com*** is running an out of date version. Please ask your server admin to update a new version of Rocket.Chat**

### PR DESCRIPTION
**Pull request by geekgonecrazy**
@RocketChat/core 

Closes #5178

Add arm support to snaps.  Many kudos to @elespike for his help on this! :+1: 

Need to test:
- [x] Upgrade from previous arm snap build
- [x] Upgrade from previous amd64 snap build

### CODE REVIEW

**Comment by geekgonecrazy** [comment]
@elespike can you take a look and see what you think?

**Comment by elespike** [comment]
@geekgonecrazy Changes look good, I didn't see any problems.

**Comment by geekgonecrazy** [comment]
Awesome!  I'll push a few more changes I made during debugging.  Then get this merged!

File: .snapcraft/resources/restoredb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+if [[ ${EUID} != 0 ]]
+then
+    echo "[-] This task must be run with 'sudo'."
+    exit
+fi
+
+backup_file=${1}
+if [[ ! -f ${backup_file} ]]
+then
+    echo "[-] Usage: snap run rocketchat-server.rcrestore ${SNAP_COMMON}/backup_file.tgz"
```
Comment (elespike) [inline .snapcraft/resources/restoredb]: @geekgonecrazy seems I overlooked this when I renamed the scripts.
Change:
echo "[-] Usage: snap run rocketchat-server.~~rcrestore~~ ${SNAP_COMMON}/backup_file.tgz"
To:
echo "[-] Usage: snap run rocketchat-server.**restoredb** ${SNAP_COMMON}/backup_file.tgz"

Comment (geekgonecrazy) [inline .snapcraft/resources/restoredb]: Ah!  I missed too!  Thanks will change

**Pull request merged**